### ISSUE TITLE
[Question] Different target in the e2e finetuning

### ISSUE THREAD

**Issue by ChenMnZ** [Cornell-RelaxML/quip-sharp#47]
Hi,

Thanks for your outstanding work! I find that QuiP# takes `orig_logits`  as the training target in the e2e finetuning process.

I want to know how much performance gain of using logit compared to using one-hot label. Do you have some detailed ablation experiments?

Thank you!

**Comment by tsengalb99** [comment]
I don't have any numbers off the top of my head but using the logits is a natural way to compare two models, whereas using one hot labels corresponds to adding a sampling layer on top of the logits.

**Comment by ChenMnZ** [comment]
Thanks for answering.
